### PR TITLE
ldap_config update

### PR DESCRIPTION
**Pull request by shampur**
Signed-off-by: Charan Shampur <cshampur@cisco.com>

typo fix

### CODE REVIEW

**Comment by shampur** [comment]
https://github.com/contiv/ccn/issues/210
https://github.com/contiv/ccn/issues/208

File: app/components/directives/settings/ldapsetting.html [hunk 1]
```diff
@@ -103,8 +87,25 @@
...
+                                <div class="field" [ngClass]="{error: service_account_password.errors?.required && formRef.submitted}">
+                                    <label for="service_account_password">Service Account Password</label>
+                                    <input #service_account_password="ngModel"
+                                           type="password"
+                                           id="service_account_password"
+                                           name="service_account_password"
+                                           [(ngModel)]="ldapConfig.service_account_password"
+                                           placeholder="Enter password" required>
+                                    <span class="inlineError" *ngIf="service_account_password.errors?.required && formRef.submitted">
+                                        Please enter ldap service account password
```
Comment (vishal-j) [inline app/components/directives/settings/ldapsetting.html]: User is forced to enter service account password whenever he edits ldap settings. I think we shouldn't make it a mandatory field. In User management, when we edit user, password is not mandatory field.

File: app/components/directives/settings/ldapsetting.html [hunk 2]
```diff
@@ -126,10 +127,25 @@
                                     </label>
                                 </div>
                             </div>
-
+                        </div>
+                        <div class="ui row" *ngIf="!ldapConfig.insecure_skip_verify && ldapConfig.start_tls">
+                            <div class="ui six wide column">
+                                <div class="field" [ngClass]="{error: tls_cert_issued_to.errors?.required && formRef.submitted}">
+                                    <label for="service_account_dn">TLS Cert issued to</label>
```
Comment (vishal-j) [inline app/components/directives/settings/ldapsetting.html]: What is TLS Cert verifier? It is not clear what to enter as value of this field from the label.

Also typo in `<label for="...">` Field id is incorrect.

File: app/login/loginctrl.ts [hunk 3]
```diff
@@ -67,7 +67,7 @@ export class LoginComponent implements OnInit{
                                 }
                             }
                         });
-                    this.chartService.startpolling();
+                    //this.chartService.startpolling();
```
Comment (vishal-j) [inline app/login/loginctrl.ts]: Remove commented code.

File: app/components/directives/settings/ldapsetting.html (hunk 1 above)
Comment (shampur) [inline app/components/directives/settings/ldapsetting.html]: Fixed

File: app/components/directives/settings/ldapsetting.html (hunk 2 above)
Comment (shampur) [inline app/components/directives/settings/ldapsetting.html]: Changed text to "Name of the server to which TLS Cert is issued"

File: app/login/loginctrl.ts (hunk 3 above)
Comment (shampur) [inline app/login/loginctrl.ts]: Fixed

File: app/components/directives/settings/ldapsetting.html [hunk 4]
```diff
@@ -140,7 +137,7 @@
                                            placeholder="Enter TLS Cert issued to"
                                            required>
                                     <span class="inlineError" *ngIf="tls_cert_issued_to.errors?.required && formRef.submitted">
-                                        Please enter TLS Cert verifier.
+                                        Please enter name of the server to which TLS Cert is issued
```
Comment (vishal-j) [inline app/components/directives/settings/ldapsetting.html]: Is it LDAP Server FQDN? Then may be we should just call it `Server FQDN`?

**Pull request merged**